### PR TITLE
[ENH] first rework of the classification notebook

### PR DESCRIPTION
**Pull request by TonyBagnall**
I've put this ready to review, but its just a starting point and all notebooks could be improved with more examples. However, I would like to get this in and then improve one at a time as we work on that type of classifier, and obviously let other people get involved. @patrickzib @hadifawaz1999 I've used your images, hope thats ok. 

part of #45

### CODE REVIEW

**Comment by hadifawaz1999** [comment]
Sounds good to me !

**Comment by patrickzib** [comment]
Sure. Fine with me.

File: examples/rocket.ipynb [hunk 1]
```diff
@@ -77,7 +77,8 @@
     "We can transform the data using ROCKET and separately fit a classifier, or we can use ROCKET together with a classifier in a pipeline (section 4, below).\n",
     "\n",
     "### 2.1 Load the Training Data\n",
-    "For more details on the data set, see the [univariate time series classification notebook](https://github.com/sktime/sktime/blob/main/examples/02_classification_univariate.ipynb)."
+    "For more details on the data set, see the [univariate time series classification\n",
+    "notebook]( ../classification.ipynb)."
```
Comment (MatthewMiddlehurst) [inline examples/rocket.ipynb:81]: Will this work? Think the full link is safer for external notebook stuff.

File: examples/01b_forecasting_proba.ipynb [hunk 2]
```diff
@@ -1606,8 +1606,8 @@
     "\n",
     "Getting started:\n",
     "\n",
-    "* follow the [\"implementing estimator\" developer guide](https://www.sktime.org/en/stable/developer_guide/add_estimators.html)\n",
-    "* use the advanced [forecasting extension template](https://github.com/sktime/sktime/blob/main/extension_templates/forecasting.py)\n",
+    "* follow the [\"implementing estimator\" developer guide](https://www.aeon.org/en/stable/developer_guide/add_estimators.html)\n",
+    "* use the advanced [forecasting extension template](https://github.com/aeon/aeon/blob/main/extension_templates/forecasting.py)\n",
```
Comment (MatthewMiddlehurst) [inline examples/01b_forecasting_proba.ipynb:1610]: Wrong GitHub link.

**Review by MatthewMiddlehurst — APPROVED** [review]
Fine to go in after these.

File: examples/rocket.ipynb (hunk 1 above)
Comment (TonyBagnall) [reply examples/rocket.ipynb:81]: I just put it in as placeholder, didnt know what the full link was!

File: examples/01b_forecasting_proba.ipynb (hunk 2 above)
Comment (TonyBagnall) [reply examples/01b_forecasting_proba.ipynb:1610]: will sort out forecasting notebooks later

**Review by MatthewMiddlehurst — APPROVED** [review]

**Pull request merged**